__tests__/fixtures/index.spec.js

```diff
@@ -1,6 +1,9 @@
 const fs = require('fs');
 const path = require('path');
 
+// eslint-disable-next-line
+const stripAnsi = require('strip-ansi');
```

Comment:
`jest` snapshots when calling update in a shell with tty will include ansi characters. By striping ansi, this will insure that tty/non-tty will work as expected for all `expect`.

Reply:
Let's consider using a [custom serializer](https://github.com/joaogranado/jest-serializer-ansi) for the snapshots

Reply:
@camacho Done.